GUI/src/javafxgui/JavaFXComponents/GUIView.java

```diff
@@ -2,17 +2,16 @@
 
 import javafx.scene.Node;
 import javafx.scene.Scene;
-import javafxgui.controller.Controller;
+import javafxgui.controller.GUIAppController;
 import jttt.core.board.Board;
 import jttt.core.board.Mark;
 import jttt.core.game.GameType;
-import jttt.core.players.Player;
 
 public class GUIView {
 
     private final int DEFAULT_BOARD_DIMENSION = 3;
     private Scene scene;
-    private Controller controller;
+    private GUIAppController controller; // IS THIS OKAY TO DO SINCE IT'S UNLIKELY ANY OTHER GUIAPP COULD BE USED ?
```

Comment:
I would still depend on the abstraction, so in general, the answer to your question is "no". What would you gain by having the dependency on the concrete type?